test/units/modules/cloud/docker/test_docker_swarm_service.py

```diff
@@ -98,3 +98,14 @@ def test_get_docker_environment(mocker, docker_swarm_service):
         None, env_files=[]
     )
     assert result == []
+
+
+def test_get_nanoseconds_from_raw_option(docker_swarm_service):
+    value = docker_swarm_service.get_nanoseconds_from_raw_option(None)
+    assert value is None
+
+    value = docker_swarm_service.get_nanoseconds_from_raw_option("1m30s500ms")
+    assert value is 90535000000
```

Comment:
```suggestion
    assert value is 90500000000
```
Or change the string to `1m30s535ms`.